### ISSUE TITLE
May I ask if you can share the pytorch code for StegaStamp?

### ISSUE THREAD

**Issue by bupt-lxc** [zhanglijun95/ZoDiac#9]
Thank you for your excellent work.

The initial Tensorflow implementation of StegaStamp is difficult to get started with. I have tried using some other pytorch implementations and they perform well against other attacks, but have difficulty decoding VAE and Diffusion attacks. Can you share the code or repository for StegaStamp?

**_Thank you again for your excellent work!_**

**Comment by zhanglijun95** [comment]
Hi, I was using the official Tensorflow implementation.

**Comment by bupt-lxc** [comment]
> Hi, I was using the official Tensorflow implementation.

Thank you again! Does CIN also use the official implementation and its original parameters? I tried to add VAE and Diffusion attack in the official implementation of CIN, but it has not been able to work. Its accuracy has been fluctuating around 0.5 and cannot be decoded. Can you share the parameters?

**Comment by zhanglijun95** [comment]
Yes I'm also using the official implementation and loading its parameters.

**Comment by Wei-Lin-Li** [comment]
May I ask if ZoDiac handles image editing as robustly as [VINE](https://github.com/Shilin-LU/VINE) (a watermarking model robust against image editing)? As I didn't find the result of ZoDiac in the VINE paper, maybe they missed this? @zhanglijun95 @bupt-lxc
